nbbuild/templates/projectized.xml

```diff
@@ -112,6 +112,9 @@ If you are sure you want to build with JDK 9+ anyway, use: -Dpermit.jdk9.builds=
                       compilecp="test.qa-functional.compile.cp"
                       compiledep="test.qa-functional.testdep"/>
         </parseprojectxml>
+        <condition property="requires.nb.javac" value="true">
+            <contains string="${module.classpath}" substring="/org-netbeans-libs-javacapi.jar" />
+        </condition>
```

Comment:
<condition property="bootclasspath.prepend" value="${bootclasspath.prepend.vanilla}">
            <istrue value="${requires.nb.javac}"/>
        </condition>
Can you please confirm if this is correct.

Reply:
The intent here is to make requires.nb.javac inferred automatically (for nb.org modules only, I think, as projectized.xml contains enhancements for nb.org modules). If a module depends on javacapi, then it should get the proper version of the API automatically (by setting the property). Not strictly necessary for this version of this patch, but feels generally useful and was needed during development.